paying_for_college/models.py

```diff
@@ -739,6 +739,65 @@ class Feedback(models.Model):
     message = models.TextField()
     url = models.TextField(blank=True, null=True)
 
+    @property
+    def parsed_url(self):
+        """parses a disclosure URL and returns a field:value dict"""
+        data = {}
+        if not self.url or 'feedback' in self.url or '?' not in self.url:
+            return data
+        split_fields = self.url.split('?')[1].split('&')
+        for field in split_fields:
+            pair = field.split('=')
+            data[pair[0]] = pair[1]
+        return data
+
+    @property
+    def school(self):
+        """Returns a school object, derived from a feedback url"""
+        if not self.url:
+            return ''
+        row = self.parsed_url
+        if row and row.get('iped'):
+            return School.objects.get(pk=row['iped'])
+        else:
+            return ''
```

Comment:
Is a `''` return here (and line 758) appropriate in place of a `School`? I don't see right off where this is called, so I'm not sure.